pkg/toolsets/mapping.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package toolsets
+
+// ToolToToolset maps tool names to their toolsets
+var ToolToToolset = map[string]Toolset{
```

Comment:
As discussed in today's call, we don't want to be so fine grained with the toolsets where we have a toolset for each API resource.

As a starting point here just so we can add support for the toolset flag, I would propose the following toolsets:

- `registry` – for workflows that want to access the public registry 
- `registry-private` – for workflows that access a private registry
- `terraform` – for workflows that need to access TFE/TFC 

In future once we've ironed out personas and their workflows, we can break the `terraform` one into smaller chunks local, admin, readonly-user, etc.